doc/manual/src/SUMMARY.md.in

```diff
@@ -26,16 +26,17 @@
 {{#include ./store/types/SUMMARY.md}}
 - [Nix Language](language/index.md)
   - [Data Types](language/values.md)
-  - [Language Constructs](language/constructs.md)
+    - [String context](language/string-context.md)
+  - [Language Syntax and Semantics](language/constructs.md)
     - [String interpolation](language/string-interpolation.md)
     - [Lookup path](language/constructs/lookup-path.md)
-    - [String context](language/string-context.md)
-  - [Operators](language/operators.md)
-  - [Derivations](language/derivations.md)
-    - [Advanced Attributes](language/advanced-attributes.md)
-    - [Import From Derivation](language/import-from-derivation.md)
-  - [Built-in Constants](language/builtin-constants.md)
-  - [Built-in Functions](language/builtins.md)
+    - [Operators](language/operators.md)
+  - [Initial Environment](language/initial-environment.md)
```

Comment:
"Environment" is one of those vague terms, unfortunately. "Initial Scope" could work, but sounds like project planning. I'm leaning towards

```suggestion
  - [Built-ins](language/builtins.md)
```

I don't like that it's a bit Nix jargon, but I don't think it's much harder to understand for newcomers, while it is much easier to understand for those who've seen `builtins`.

Reply:
Hm. `language/builtins.md` is currently the name of the ‘Built-in Functions’ section, which I was planning on keeping more or less as-is (seemed like there was intention behind the choice to split that and `builtin-constants.md` into different files). My intent with this ‘Initial Environment’ section was to list the identifiers bound in the, uh, initial environment, and link to the appropriate anchor in `builtins.md` or `builtin-constants.md` for reference documentation on each of them.

What would you think of ‘Top-Level Scope’?

Reply:
`builtins.md` and `builtins-constants.md` should be combined. There is no user-facing difference between them (because a constant like `builtins.derivation` can also be a function), just internal implementation details which we should not leak.

So given that they should be combined, I think @roberth's suggestion is a good one --- we don't need a 3rd page linking to one of these 2 pages, we just need one page that documents `builtins` and points out which ones are also available in the top-level scope.

Reply:
> `builtins.md` and `builtins-constants.md` should be combined.

Well if that's the case then I'm happy to oblige—that makes the most sense to me as well.

Should I perhaps create a new PR to do that? I think it'll touch rather different things from this one, and might be more digestible as its own unit.